### DISCUSSION TITLE
Adjustable window sizes when viewing a recording/events

### DISCUSSION THREAD

**Discussion by McPaHa** [blakeblackshear/frigate#13912] [Ideas]
Hi!

I would like to suggest a feature improvement for the event/recording viewer. Currently, I find it inconvenient that the selected camera is always displayed in a large window while the others are minimized.

I would appreciate the ability to:

**Customize the layout:** Choose which cameras are displayed and adjust the size of their respective windows.

**Implement a unified timeline:** Display a timeline that shows events from all selected cameras simultaneously.

This would significantly enhance the user experience by allowing for a more comprehensive overview of the event, without the need to constantly switch between camera views.

**Comment by NickM-27** [comment]
you can already do all of this with the motion tab in review.

There is no point in displaying recordings when the view is going to be so much smaller that the additional resolution nets no benefit
